README.md

```diff
@@ -31,12 +32,13 @@ Start using the application today to quickly create, manage, and view your IoT d
    ```sh
    yarn install
    ```
+1. Update the variables `CLIENT_AWS_ACCESS_KEY_ID`, `CLIENT_AWS_SECRET_ACCESS_KEY`, and `CLIENT_AWS_SESSION_TOKEN` under file `apps/core/.env` with your development AWS credentials.
 1. Start development server:
    ```sh
    yarn dev
    ```
 1. Application is available at URL: `http://localhost:3000`
-1. Log in with local Cognito credentials found at `apps/core/.cognito/db`
+1. Log in with local Cognito credentials found at `apps/core/.cognito/db/us-west-2_h23TJjQR9.json`
```

Comment:
will this file name be dependent on the region of the AWS credentials?

Reply:
No, All static for local dev purpose only.